shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -612,11 +615,34 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   if ([call.method isEqualToString:@"SystemNavigator.pop"]) {
     [NSApp terminate:self];
     result(nil);
+  } else if ([call.method isEqualToString:@"Clipboard.getData"]) {
+    result([self getClipboardData:call.arguments]);
+  } else if ([call.method isEqualToString:@"Clipboard.setData"]) {
+    [self setClipboardData:call.arguments];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+#pragma mark - ClipboardData
+
+- (NSDictionary*)getClipboardData:(NSString*)format {
```

Comment:
These methods need to be declared and commented in the private interface declarations section near the top of the file, per project style.